RecoLocalMuon/CSCRecHitD/python/customRecHitBuilder.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+def tightenAnodeTimes(process):
+    if hasattr(process,'csc2DRecHits'):
+        process.csc2DRecHits.CSCUseReducedWireTimeWindow = cms.bool(True)
+        process.csc2DRecHits.CSCWireTimeWindowLow = cms.int32(5)
+        process.csc2DRecHits.CSCWireTimeWindowHigh = cms.int32(11)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
So, ```process.csc2DRecHits.CSCUseReducedWireTimeWindow = True```